test/integration/shoots/operations/shoot_op_test.go

```diff
@@ -0,0 +1,159 @@
+			Expect(err).To(BeNil())
+
+			var wg sync.WaitGroup
+			for _, shootToDelete := range shoots {
+				if strings.HasPrefix(shootToDelete.Name, framework.IntegrationTestPrefix) {
+					wg.Add(1)
+					go func(shootDel v1beta1.Shoot) {
+						defer wg.Done()
+						shootGardenerCleanup, err := framework.NewShootGardenerTest(*kubeconfig, &shootDel, shootOperationsTestLogger)
+						Expect(err).To(BeNil())
+
+						timeoutCtx, cancel := context.WithTimeout(context.Background(), WaitForShootDeletionTimeout)
+						defer cancel()
+						shootDeletion := context.WithValue(timeoutCtx, "name", "delete shoot")
+
+						err = shootGardenerCleanup.DeleteTestShoot(shootDeletion)
+						Expect(err).To(BeNil())
+
+					}(*shootToDelete)
+				}
+			}
+			wg.Wait()
+			shootGardenerTest.Logger.Info("Clean up complete!")
+		}
+	})
+
+	// Create only test
+	//It("should create shoot successfully", func() {
+	//	timeoutCtx, cancel := context.WithTimeout(context.Background(), WaitForShootCreationTimeout)
+	//	defer cancel()
+	//
+	//	shootCreationCtx := context.WithValue(timeoutCtx, "name", "create shoot")
+	//	_, err := shootGardenerTest.CreateTestShoot(shootCreationCtx)
+	//	Expect(err).NotTo(HaveOccurred())
+	//})
+
+	//// Delete only test: needs the name of the shoot <as a cli flag) to delete as well as the namespace it resides on <as a cli flag>. Should you choose
+	//// not to run this test, please ignore it or comment it out.
+	//It("should delete shoot successfully", func() {
+	//	timeoutCtx, cancel := context.WithTimeout(context.Background(), WaitForShootDeletionTimeout)
+	//	defer cancel()
+	//	shootDeletion := context.WithValue(timeoutCtx, "name", "create shoot")
+	//
+	//	if len(*shootName) == 0 || len(*shootNamespace) == 0 {
+	//		// Raise an error to make test fail gracefully if shoot does not exist
+	//		Fail(framework.ErrShootsNamespaceWasNotSpecified.Error())
+	//	}
+	//	shootOperationsTestLogger.Infof("Testing shoot deletion for %s", *shootName)
+	//	shootGardenerTest.Shoot = &v1beta1.Shoot{
+	//		ObjectMeta: metav1.ObjectMeta{
+	//			Name:      *shootName,
+	//			Namespace: *shootNamespace,
+	//		},
+	//	}
+	//
+	//	err := shootGardenerTest.DeleteTestShoot(shootDeletion)
+	//	Expect(err).To(BeNil())
+	//})
+	//
+	// This test creates and deletes shoots all together
+	FIt("should create and delete shoot successfully", func() {
+		// First we create the target shoot.
+		timeoutCtx, cancel := context.WithTimeout(context.Background(), WaitForCreateDeleteTimeout)
+		defer cancel()
+		shootCreateDelete := context.WithValue(timeoutCtx, "name", "create and delete shoot")
+
+		_, err := shootGardenerTest.CreateTestShoot(shootCreateDelete)
+		Expect(err).NotTo(HaveOccurred())
+		// Now we should test shoot deletion
+		shootOperationsTestLogger.Infof("Testing shoot deletion for %s", shoot.Name)
+		// Then we delete the shoot
+		err = shootGardenerTest.DeleteTestShoot(shootCreateDelete)
+		if err != nil {
+			shootOperationsTestLogger.Error(err.Error())
```

Comment:
If we're integrating into Ginkgo properly, you should use its logger as described in https://onsi.github.io/ginkgo/#logging-output